user-guide/Advanced_Functionality/Databases/Elasticsearch_database/Installing_Elasticsearch_on_Linux.md

```diff
@@ -0,0 +1,116 @@
+---
+uid: Installing_Elasticsearch_on_Linux
+---
+
+# Installing Elasticsearch on a Linux machine
+
+If you want to use an Elasticsearch cluster for your DMS (which is required to use the [Cassandra Cluster](xref:Migrating_the_general_database_to_a_DMS_Cassandra_cluster) feature), install Elasticsearch on a Linux machine as detailed below.
+
+1. Install the Elasticsearch software on  the Linux machine as described under [Installing from the RPM repository](https://www.elastic.co/guide/en/elasticsearch/reference/6.8/rpm.html#rpm-repo) in the official Elasticsearch documentation.
+
+1. Make sure the firewall ports are open for Elasticsearch. Elasticsearch operates on TCP port 9200 and TCP port 9300.
+
+   - There is a default firewall on Linux, but this is disabled by default. To enable the firewall, use the following command:
+
+     `$ systemctl start firewalld.service`
```

Comment:
Firewalld is not installed by default on Ubuntu (Debian), invalidating the entire section. 
There might be something else, but this would need to be looked into further.